06-Customizing AzTS for your org/Extending AzTS/ManagementGroupBasedComplianceInitiative.md

```diff
@@ -67,6 +67,34 @@ By default, this feature is not turned on. If you haven't enabled it in your AzT
 10. Click 'OK' and then click 'Save'.
 ![Resources](../../Images/06_MGCIEnableMGAndSave.png)
 
+-----
+
+- Using helper script
```

Comment:
The case for "Using helper script" should be at level with other headings.
Also, what is this an alternate solution to? Step 1 or 2 in "Steps to enable Management Group driven Compliance Initiative feature" or is it an alternative to all the steps?

It should be placed appropriately and we should use "Option 1"/"Option 2" along with "Using helper script" and again, appropriate casing

Reply:
The above script is an  alternate solution to Step1 and 2. Added Option1 and Option 2 in sub-heading